CefSharp.OutOfProcess.BrowserProcess/BrowserProcessHandler.cs

```diff
@@ -87,13 +87,23 @@ Task IOutOfProcessClientRpc.CloseHost()
             });
         }
 
-        Task IOutOfProcessClientRpc.CreateBrowser(IntPtr parentHwnd, string url, int id)
+        Task IOutOfProcessClientRpc.CreateBrowser(IntPtr parentHwnd, string url, int id, IDictionary<string, object> requestContextPreferences)
         {
             //Debugger.Break();
 
             return CefThread.ExecuteOnUiThread(() =>
             {
-                var browser = new OutOfProcessChromiumWebBrowser(_outOfProcessServer, id, url);
+                IRequestContext requestContext = null;
+                if (requestContextPreferences != null)
+                {
+                    requestContext = new RequestContext();
```

Comment:
Assigning a new `RequestContext` here dramatically changes the behaviour, you've not provided any way to manage the `RequestContextSettings`, so you'll always end up with an in memory cache.

This needs a rethink.

Reply:
would it use the previous configured cefsettings if i use the globalrequestcontext values? 
and therefore use the cachePath set there